arm-dependencies.sls

```diff
@@ -0,0 +1,78 @@
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+/home/servo/bin:
```

Comment:
I think we should add some subdirectories inside this folder to keep the arm32 and arm64 binaries cleanly separated.